### PR TITLE
SOC-guidelines

### PR DESCRIPTION
**Pull request by JurasJuras**
Created six tests for Discussions Guidelines, @nikodamn , @mleczey Please CR

### CODE REVIEW

**Comment by WikiaTech** [comment]
SonarQube analysis reported 2 issues:
* MAJOR 2 major

Watch the comments in this conversation to review them.
Note: the following issues could not be reported as comments because they are located on lines that are not displayed in this pull request:
* MAJOR [2 duplicated blocks of code must be removed.](https://github.com/Wikia/selenium-tests/blob/a5fa477f99ed26ece25bf410f40214feba6ee1a4/src/test/java/com/wikia/webdriver/elements/mercury/desktop/PostsCreatorDesktop.java) [rule](http://sonar-s1/coding_rules#rule_key=common-java%3ADuplicatedBlocks)

File: src/test/java/com/wikia/webdriver/elements/mercury/desktop/BackButtons.java [hunk 1]
```diff
@@ -1,4 +1,4 @@
-package com.wikia.webdriver.elements.mercury.components.discussions.desktop;
```
Comment (mleczey) [inline src/test/java/com/wikia/webdriver/elements/mercury/desktop/BackButtons.java]: Why did you change packages for all files in this pull request?

File: src/test/java/com/wikia/webdriver/elements/mercury/pages/discussions/GuidelinesPage.java [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+
+    public GuidelinesPage open() {
+        driver.get(urlBuilder.getUrlForWiki() + String.format(PATH));
+        return this;
+    }
+
+    public DiscussionsPage clickBackToDiscussions() {
+        wait.forElementClickable(backToDiscussionsButton);
+        backToDiscussionsButton.click();
+
```
Comment (mleczey) [inline src/test/java/com/wikia/webdriver/elements/mercury/pages/discussions/GuidelinesPage.java]: Shouldn't you wait here for discussions page to be loaded?

File: src/test/java/com/wikia/webdriver/testcases/discussions/NavigatingTests.java [hunk 3]
```diff
@@ -55,8 +55,7 @@ public void anonUserOnDesktopCanClickAvatar() {
   @Test(groups = "discussions-anonUserOnDesktopCanClickUsername")
   @Execute(asUser = User.ANONYMOUS)
   @InBrowser(browser = Browser.FIREFOX, browserSize = DESKTOP_RESOLUTION)
-  public void anonUserOnDesktopCanClickUsername() {
-    clickUsernameLoadsUserPage();
```
Comment (mleczey) [inline src/test/java/com/wikia/webdriver/testcases/discussions/NavigatingTests.java]: Revert this style change.

**Comment by PiotrGackowski** [comment]
Consider adding entry to 
selenium-tests\src\test\java\com\wikia\webdriver\testsuites\testSuite.xml

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue:
* MAJOR 1 major

Watch the comments in this conversation to review them.
